src/ServiceManagement/Compute/Compute.Tests/SessionRecords/Microsoft.WindowsAzure.Management.Compute.Testing.ComputeOperationsTests/CanAbortVirtualMachineDeployment.json

```diff
@@ -13,7 +13,7 @@
           "214"
         ],
         "x-ms-version": [
-          "2016-06-01"
+          "2017-01-01"
```

Comment:
talked to pengfei offline, since there may be behavioral changes with the new rdfe api version we will go with the major version update